render-release-notes.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
```

Comment:
Let's use `/bin/sh`... no need to fall prey to bash'isms (which would make contributing to Git's shell scripts hard, as they are required to be free of Bash's idiosyncrasies).

Also: could you sign-off this commit?